### PR TITLE
Add extra div to make agents table start from sticky header

### PR DESCRIPTION
**Pull request by iamsumanth**
#2855

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/agents/agents_widget.js.msx [hunk 1]
```diff
@@ -233,6 +233,7 @@ define([
 
       return (
         <div onclick={ctrl.hideDropDownsAndMessage.bind(ctrl)}>
+          <div class="extra-div-to-replace-sticky-header-height" style={!s.isBlank(message) ? "height:284px;" : ''}></div>
```
Comment (zabil) [inline server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/agents/agents_widget.js.msx]: Do not use styles, add or remove classes, 
move this method to controller.

```extra-div-to-replace-sticky-header-height``` is too specific call it ```agent-header-spacer```

File: server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_common.scss [hunk 2]
```diff
@@ -29,6 +29,9 @@ body {
 
 .main-container {
   margin-top: 50px;
+  #agents-page & {
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_common.scss]: File says common.scss, please move this into the agents page.

File: server/webapp/WEB-INF/rails.new/app/views/layouts/single_page_app.html.erb [hunk 3]
```diff
@@ -13,7 +13,7 @@
   <%= csrf_meta_tags %>
 </head>
 
-<body data-controller-name="<%= controller_name %>" data-action-name="<%= action_name %>">
+<body data-controller-name="<%= controller_name %>" data-action-name="<%= action_name %>" id="<%= controller_name %>-page">
```
Comment (ketan) [inline server/webapp/WEB-INF/rails.new/app/views/layouts/single_page_app.html.erb]: This id is not needed, the data attributes are sufficient selectors.

**Comment by iamsumanth** [comment]
This fix is asserted as a part of https://github.com/gocd/gocd/pull/2809 pull request.

Can someone please close it.

**Pull request closed without merging**